mora/settings.py

```diff
@@ -33,6 +33,8 @@
 SSO_SAML_METADATA_URL = '/url/to/sso/metadata'
 SSO_SAML_USERNAME_ATTR = ''
 
+SAML_IDP_INSECURE = True
```

Comment:
You're overwriting the [previous assignment](https://github.com/magenta-aps/mora/pull/134/commits/b37409d1faa9a3f24579476243c5d66cbeebeecf#diff-99faeed6f80f83309c4eb1d6f068d108R30) to this variable.